### ISSUE TITLE
Add support for node 15

### ISSUE THREAD

**Issue by Powersource** [Edgeryders-Participio/realities#148]
I updated my system to node 15 apparently and now building doesn't want to work lol. I tried updating a couple of packages but that didn't seem to be enough, the rabbit hole is deeper

**Comment by Powersource** [comment]
hmm lol ok it seems to build on master :shrug: but doesn't hurt updating stuff anyway

**Comment by Powersource** [comment]
fun https://github.com/facebook/create-react-app/issues/9986

**Comment by nhevia** [comment]
Commenting here because 9986 is unrelated to node versioning.  
You should use v14, which is the `active` version. v15 just hit `current` status, which means 6 months support for maintainers to test their libraries in a "pre v16" environment.

[Releases](https://nodejs.org/en/about/releases/)  

![imagen](https://user-images.githubusercontent.com/50466554/99098669-9bfb4a00-25d9-11eb-8e1b-16778636ce00.png)

**Comment by Powersource** [comment]
Yeah we should probably keep using 14 in prod. But I'm the kind of fool who uses arch so I end up with 15 locally. I'd prefer not to have to use nvm since its ux is pretty bad but I'll consider it.

**Comment by 0xR** [comment]
You could try: https://github.com/Schniz/fnm
much faster than nvm

I'm also using `eval "$(fnm env --multi --use-on-cd)"`

makes me automatically get the right node version everywhere
